src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random random = new Random();
+    private static Color color;
+
+    public static Ball getRandomBall() {

```

Comment:
Usage of static methods is prohibited by task requirements
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-use-static-methods-in-your-solution
```suggestion
    public Ball getRandomBall() {

```